python/l1tPh2Nanotables_cff.py

```diff
@@ -243,6 +243,47 @@
     doc = cms.string("GMT Tk Muons, origin: GMT"),
 )
 # gmtTkMuTable.variables.nStubs = Var("stubs().size()",int,doc="number of stubs")
+### Standalone Muon from GMT, before ghost busting
+
+KMTFpromptMuTable = staMuTable.clone(
+    src = cms.InputTag("l1tKMTFMuonsGmt", "prompt"),
+    name = cms.string("L1KMTFMuon"),
```

Comment:
could you put the `XXTF` labels at the end? this will harmonize the naming and make it easier to read

Reply:
For the name? Can you show the example here so that I can update it?

Reply:
`name = cms.string("L1MuonKMTF"),` would be my proposal

Reply:
Ok, done